.drone.yml

```diff
@@ -35,14 +35,23 @@ pipeline:
   #     branch: master
 
   publish:
-    image: plugins/docker
-    repo: drone/drone
-    username: ${DOCKER_USERNAME}
-    password: ${DOCKER_PASSWORD}
-    tag: [ 0.6 ]
-    when:
-      branch: master
-      event: push
+
+    docker:
```

Comment:
Ah, I was going by the 0.4 documentation instead of the newer stuff. That could really use a clarification. I'll get that fixed up.